acme/tests/client_test.py

```diff
@@ -705,8 +710,10 @@ def test_fetch_chain_too_many(self):  # recursive
 
     def test_revoke(self):
         self.client.revoke(self.certr.body, self.rsn)
-        self.net.post.assert_called_once_with(
-            self.directory[messages.Revocation], mock.ANY, acme_version=1)
+        with warnings.catch_warnings():
+            warnings.filterwarnings('ignore', '.* non-string keys', DeprecationWarning)
+            self.net.post.assert_called_once_with(
+                self.directory[messages.Revocation], mock.ANY, acme_version=1)
```

Comment:
We'll have to do something like this long term right? If so, I think we should just do it now.
```suggestion
        self.net.post.assert_called_once_with(
            self.directory['revoke-cert'], mock.ANY, acme_version=1)
```